Include CoreFoundation instead of Carbon since this code only uses
CoreFoundation stuff, and including Carbon plus "using namespace boost"
causes "error: reference to 'Collection' is ambiguous".
--- src/Glob2.cpp.orig	2009-08-29 15:39:06.000000000 -0500
+++ src/Glob2.cpp	2018-10-17 21:37:30.000000000 -0500
@@ -62,7 +62,7 @@
 #endif
 
 #ifdef __APPLE__
-#	include <Carbon/Carbon.h>
+#	include <CoreFoundation/CoreFoundation.h>
 #	include <sys/param.h>
 #endif
 
